code/modules/reagents/reagents/stims.dm

```diff
@@ -4,6 +4,8 @@
 	constant_metabolism = TRUE
 	reagent_type = "Stimulator"
 
+	var/sanity_gain
```

Comment:
You should place this variable under `/datum/reagent`.

Reply:
oh thanks. (i was indeed unsure on where to insert it. thanks for correcting my deadbrain stuff) will do right now